features/step_definitions/journey_test_steps.rb

```diff
@@ -0,0 +1,340 @@
+# encoding: utf-8
+require "ostruct"
+require "rest_client"
+
+store = OpenStruct.new
+store.changed_fields = store.changed_fields || Hash.new
+
+Given /I am on the '(.*)' login page$/ do |user_type|
+  if user_type == 'Administrator'
+    visit("#{dm_admin_frontend_domain}/admin/login")
+    page.should have_content("#{user_type}" ' login')
+    page.should have_content('Username')
+    page.should have_content('Password')
+  else user_type == 'Suppliers'
+    visit("#{dm_supplier_frontend_domain}/#{user_type.downcase}/login")
+    page.should have_content('Email address')
+    page.should have_content('Passphrase')
+    page.should have_content('Login')
+  end
+end
+
+When /I login as a '(.*)' user$/ do |user_type|
+  if user_type == 'Administrator'
+    fill_in('username', :with => ENV['DM_ADMINISTRATORNAME'])
+    fill_in('password', :with => ENV['DM_ADMINISTRATORPASSWORD'])
+    click_link_or_button('Log in')
+  else user_type == 'Supplier'
+    fill_in('email-address', :with => ENV['DM_SUPPLIEREMAIL'])
+    fill_in('passphrase', :with => ENV['DM_SUPPLIERPASSPHRASE'])
+    click_link_or_button('Login')
+    page.should have_content('Dashboard')
+  end
+end
+
+Then /I am presented with the '(.*)' page$/ do |page_name|
+  page.should have_content(page_name)
+  page.should have_content('Service ID')
+end
+
+When /I enter '(.*)' in the '(.*)' field$/ do |value,field_name|
+  fill_in(field_name, with: value)
+  store.serviceID = value
+end
+
+And /I click the '(.*)' button$/ do |button_name|
+  click_link_or_button(button_name)
+end
+
+Then /I am presented with the summary page for that service$/ do
+  store.existing_values = store.existing_values || Hash.new
+  store.existing_values['summarypageurl'] = current_url
+
+  current_url.should have_content(store.serviceID)
+  page.should have_content('Service attributes')
+  page.should have_content('Description')
+  page.should have_content('Features and benefits')
+  page.should have_content('Pricing')
+  page.should have_content('Documents')
+
+  servicename = find(
+    :xpath,
+    "//table[2]/tbody/tr[1]/td[2]"
```

Comment:
This drilling down through elements seems quite fragile. Not sure if that's good or bad, maybe something we should talk about as a group?